Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<h1>Part 5. Assets, <em>Data Views, </em>and Metadata</h1><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done